examples/nlp/language_modeling/tuning/conf/megatron_gpt_adapter_tuning_config.yaml

```diff
@@ -103,9 +103,12 @@ model:
     answer_field: "target"
 
   adapter_tuning:
+    type: 'parallel'
     adapter_dim: 50
     adapter_dropout: 0.1
+    column_init: 'xavier'
     norm_position: 'pre' # This can be set to 'pre' or 'post', 'pre' is normally what is used.
+    norm_type: 'mixedfused'
```

Comment:
This is a generic prefix for both layernorm and rmsnorm can it be more verbose?

Reply:
We use the same argument name elsewhere to provide either `layernorm` or `rmsnorm` as possible values. This will probably who use both confused. Can you also include the list of acceptable values for this arg?

Reply:
Updated to more specific option names.